AlphaWallet/Settings/ViewModels/AdvancedSettingsViewModel.swift

```diff
@@ -11,10 +11,11 @@ import UIKit
 struct AdvancedSettingsViewModel {
     var rows: [AdvancedSettingsRow] = {
         let privateNerworkRow: [AdvancedSettingsRow] = Features.isUsingPrivateNetwork ? [.usePrivateNetwork] : []
+        let analyticsRow: [AdvancedSettingsRow] = Features.isAnalyticsUIEnabled ? [.analytics] : []
         if Features.isLanguageSwitcherDisabled {
-            return [.console, .clearBrowserCache, .tokenScript, .pingInfura] + privateNerworkRow
+            return [.console, .clearBrowserCache, .tokenScript, .pingInfura] + privateNerworkRow + analyticsRow
         } else {
-            return [.console, .clearBrowserCache, .tokenScript, .changeLanguage, .pingInfura] + privateNerworkRow
+            return [.console, .clearBrowserCache, .tokenScript, .changeLanguage, .pingInfura] + privateNerworkRow + analyticsRow
         }
```

Comment:
Maybe we rewrite this section? Because there will be more and more Features.isLanguageSwitcherDisabled, Features.isExportJsonKeystoreEnabled, etc. @hboon 

```swift
    var rows: [AdvancedSettingsRow] = {
        return [
            .console,
            .clearBrowserCache,
            .tokenScript,
            .pingInfura,
            Features.isUsingPrivateNetwork ? .usePrivateNetwork : nil,
            Features.isAnalyticsUIEnabled ? .analytics : nil,
            Features.isLanguageSwitcherDisabled ? nil : .changeLanguage,
        ].compactMap { $0 }
    }()
```

Reply:
Sure